nobuco/node_converters/tensor_getattr.py

```diff
@@ -0,0 +1,21 @@
+from typing import Optional
+
+import torch
+from torch import Tensor
+
+import tensorflow as tf
+
+from nobuco.commons import ChannelOrderingStrategy
+from nobuco.converters.node_converter import converter
+
+
+@converter(torch.Tensor.__getattribute__, channel_ordering_strategy=ChannelOrderingStrategy.FORCE_PYTORCH_ORDER)
```

Comment:
For efficiency reasons, `channel_ordering_strategy` should be `MINIMUM_TRANSPOSITIONS`